install.sh

```diff
@@ -0,0 +1,71 @@
+if command -v python3 >/dev/null 2>&1 ; then
+    echo "python3 found"
+    echo "version: $(python3 --version)"
+else
+    echo "python3 not found, please install it"
+    exit
+fi
+
+if command -v pip3 >/dev/null 2>&1 ; then
+    echo "pip3 found"
+else
+    echo "pip3 not found, please install it"
+    exit
+fi
+
+if command -v virtualenv >/dev/null 2>&1 ; then
+    echo "virtualenv found"
+else
+    echo "virtualenv not found, please install it"
+    pip3 install virtualenv
+fi
+
+if [ -d ~/.envs/APIPerf3.5 ]; then
+  echo "~/.envs/APIPerf3.5 found"
+else
+  echo "~/.envs/APIPerf3.5 not found"
```

Comment:
better move this to after creating the new one  and say you solved the problem :)